CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.10.0
+
+_Jul 11, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add selectors to support showing child row count in footer in the Data Grid
+- 🧰 Add utility function to check if an optional plugin is present in the Tree View
```

Comment:
I'm not sure we need to highlight this mainly internal addition..? 🤔 
cc @flaviendelangle @noraleonte

Reply:
Hmm, I don't think so. It's an internal thing 🤔

Reply:
```suggestion
```